### ISSUE TITLE
some question about the results

### ISSUE THREAD

**Issue by suny105** [Chiangsonw/CaLa#6]
Hi! Thanks for you interesting work. I meet some gaps when i reproduce the cirr dataset. Here are my config and result:
`{
    "batch_size": 64,
    "cir_frame": "sum",
    "embeds_dim": 768,
    "encoder": "multi",
    "encoder_arch": "blip2",
    "experiment_name": "019a7730-f32d-7ccf-8941-151459e2f8d1",
    "hca_temperature": 0.92,
    "hca_weight": 0.1,
    "learning_rate": 5e-06,
    "max_epoch": 15,
    "min_lr": 0.0,
    "model_name": "blip2_feature_extractor",
    "num_epochs": 30,
    "save_best": true,
    "save_training": true,
    "tac_temperature": 2.3,
    "tac_weight": 0.45,
    "target_ratio": 1.25,
    "transform": "targetpad",
    "validation_frequency": 1
}`
Result:

Recall@ | Score (Accuracy Perc.)
-- | --
1 | 42.289
2 | 56.506
5 | 73.373
10 | 84.024
50 | 95.976

RecallSubset@ | Score (Accuracy Perc.)
-- | --
1 | 72.193
2 | 88.241
3 | 94.627

I'd like to know if I've made a mistake somewhere.
Thanks!

**Comment by Chiangsonw** [comment]
Try to set learning rate to 1e-5, you can reference to an earlier issue
